src/localization/en-CA.json

```diff
@@ -0,0 +1,150 @@
+  "checkout-summary": "Checkout Summary",
+  "complete-order": "Complete Order",
+  "contact-us": "Contact Us",
+  "contact-us-content": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+  "something-went-wrong-message": "Oops! Something went wrong...",
+  "try-again-message": "Please go back and try again or",
+  "return-home": "return home",
+  "name-moki": "Moki",
+  "purchase-details": "Purchase Details",
+  "shipping-option": "Shipping Option",
+  "shipping-address": "Shipping Address",
+  "review-your-order": "Review Your Order",
+  "complete-purchase": "Complete Purchase",
+  "profile": "Profile",
+  "order-processed-message": "Thank you! Your order has been processed.",
+  "terms-and-conditions": "Terms &amp; Conditions",
+  "terms-and-conditions-content": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+  "province": "Province",
+  "address": "Address",
+  "new" : "New",
+  "first-name": "First Name",
+  "last-name": "Last Name",
+  "street-address": "Street Address",
+  "extended-address": "Extended Address",
+  "country": "Country",
+  "postal-code": "Postal Code",
+  "save": "Save",
+  "cancel": "Cancel",
+  "vestri": "Vestri",
+  "store-description-message": "Commerce software that powers the next generation of digital experience",
+  "assistance": "Assistance",
+  "contact": "Contact",
+  "find-us-online": "Find us Online",
+  "facebook": "facebook",
+  "twitter": "twitter",
+  "instagram": "instagram",
+  "language": "Language",
+  "currency": "Currency",
+  "logout": "Logout",
+  "login": "Login",
+  "categories": "Categories",
+  "search": "Search",
+  "in-stock": "In Stock",
+  "pre-order": "Pre-order",
+  "back-order": "Back-order",
+  "out-of-stock": "Out of Stock",
+  "expected-release-date": "Expected Release Date",
+  "remove": "Remove",
+  "shopping-card-empty-message": "Your shopping cart is empty.",
+  "product": "Product",
+  "name": "Name",
+  "options": "Options",
+  "availability": "Availability",
+  "unit-price": "Unit Price",
+  "quantity": "Quantity",
+  "total-price": "Total Price",
+  "no-products-found": "No products found",
+  "todays-discount": "Today&apos;s Discount",
```

Comment:
These "&apos;" don't appear to be getting rendered anymore. With the new lib, can we replace them now with **'**?
![image](https://user-images.githubusercontent.com/39800563/43647878-eba0182e-9706-11e8-901e-4f2b471f1e99.png)

Reply:
You're right! Same goes for `&amp;`